person.rb

```diff
@@ -1,14 +1,16 @@
 require_relative './student'
 require_relative './teacher'
+require_relative './nameable'
 
-class Person
+class Person < Nameable 
     extend Student
     extend Teacher
 
     attr_reader :id
     attr_writer :name, :age, :parent_permission
 
-    def initializing (age, name: 'Unknown', parent_permission: true)
+    def initializing (age, name, parent_permission: true)
```

Comment:
- [ ] After making Nameable a class, I am still not able to run your code, I get this error ```wrong number of arguments (given 2, expected 1) (ArgumentError)```. I suggest that instead of ```def initializing```, you make it ```def initialize``` and remove the space before the parentheses.